### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by freddy1020**
#### What is it?
changes in README.md

#### Description of the changes in your PR
Fixed several errors in the document to make it easy to understand.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,15 +97,15 @@ You can install NewPipe using one of the following methods:
...
+We recommend method 1 for most users. APKs installed using method 1 or 2 are compatible with each other (meaning that if you installed NewPipe using either method 1 or 2, you could also update NewPipe using the other), but not with those installed using method 3. This is because the same signing key (ours) is used for 1 and 2, but a different signing key (F-Droid's) is used for 3. Building a debug APK using method 4 excludes a key entirely. Signing keys help ensure that a user isn't tricked into installing a malicious update to an app. When using method 5, each APK is signed with a different random key supplied by GitHub Actions, so you cannot even update it. You must backup and restore the app data each time you wish to use a new APK.
```
Comment (TacoTheDank) [inline README.md:100]: I think I prefer "You will have to backup" over "You must backup."

File: README.md [hunk 2]
```diff
@@ -97,15 +97,15 @@ You can install NewPipe using one of the following methods:
...
-1. Back up your data via Settings > Content > Export Database so you keep your history, subscriptions, and playlists
+1. Back up your data via Settings > Content > Export Database, so you keep your history, subscriptions, and playlists
 2. Uninstall NewPipe
 3. Download the APK from the new source and install it
 4. Import the data from step 1 via Settings > Content > Import Database
 
-<b>Note: when you're importing a database into the official app, always make sure that it is the one you exported _from_ the official app. If you import a database exported from an APK other than the official app, it may break things. Such an action is unsupported, and you should only do so when you're absolutely certain you know what you're doing.</b>
+<b>Note: when you're importing a database into the official app, always make sure that it is the one you exported _from_ the official app. If you import a database exported from an APK other than the official app, it may break things. Such action is unsupported, and you should only do so when you're absolutely certain you know what you're doing.</b>
```
Comment (TacoTheDank) [inline README.md:108]: Same here, I think "Such an action" is better.

**Review by TacoTheDank — COMMENTED** [review]
I think these are generally good changes, I just have some comments.

File: README.md (hunk 1 above)
Comment (opusforlife2) [reply README.md:100]: 'Must' is a recommendation. 'Will have to' is a requirement. If the user doesn't want to lose app data, this step is required.

File: README.md (hunk 2 above)
Comment (opusforlife2) [reply README.md]: The suggested change is wrong.

**Comment by freddy1020** [comment]
Yeah, I have recovered these two changes.

File: README.md [hunk 3]
```diff
@@ -53,13 +53,13 @@ As you can see, NewPipe supports multiple video and audio services. Though it st
 
 Partially due to circumstance, and partially due to its popularity, YouTube is the best supported out of these services. If you use or are familiar with any of these other services, please help us improve support for them! We're looking for maintainers for SoundCloud and PeerTube.
 
-If you intend to add a new service, please get in touch with us first! Our [docs](https://teamnewpipe.github.io/documentation/) provide more information on how a new service can be added to the app and to the [NewPipe Extractor](https://github.com/TeamNewPipe/NewPipeExtractor).
+If you intend to add a new service, please get in touch with us first! Our [docs](https://teamnewpipe.github.io/documentation/) provide more information on how a new service can be added to the app and the [NewPipe Extractor](https://github.com/TeamNewPipe/NewPipeExtractor).
```
Comment (opusforlife2) [inline README.md:56]: Adds ambiguity. -1

File: README.md [hunk 4]
```diff
@@ -53,13 +53,13 @@ As you can see, NewPipe supports multiple video and audio services. Though it st
...
 
 ## Description
 
-NewPipe works by fetching the required data from the official API (e.g. PeerTube) of the service you're using. If the official API is restricted (e.g. YouTube) for our purposes, or is proprietary, the app parses the website or uses an internal API instead. This means that you don't need an account on any service to use NewPipe.
+NewPipe works by fetching the required data from the official API (e.g. PeerTube) of the service you're using. If the official API is restricted (e.g. YouTube) for our purposes, or is proprietary, the app parses the website or uses an internal API instead. This means you don't need an account on any service to use NewPipe.
```
Comment (opusforlife2) [inline README.md:60]: Both mean the same thing. I prefer less ambiguity. -1

File: README.md [hunk 5]
```diff
@@ -53,13 +53,13 @@ As you can see, NewPipe supports multiple video and audio services. Though it st
...
 
-Also, since they are free and open source software, neither the app nor the Extractor use any proprietary libraries or frameworks, such as Google Play Services. This means you can use NewPipe on devices or custom ROMs that do not have Google apps installed.
+Also, since they are free and open source software, neither the app nor the Extractor uses any proprietary libraries or frameworks, such as Google Play Services. This means you can use NewPipe on devices or custom ROMs that do not have Google apps installed.
```
Comment (opusforlife2) [inline README.md:62]: Normally, I would agree, but now it breaks number agreement/consistency with "since they are" and that seems wrong. @mhmdanas?

File: README.md [hunk 6]
```diff
@@ -97,10 +97,10 @@ You can install NewPipe using one of the following methods:
...
+We recommend method 1 for most users. APKs installed using method 1 or 2 are compatible with each other (meaning that if you installed NewPipe using either method 1 or 2, you could also update NewPipe using the other), but not with those installed using method 3. This is because the same signing key (ours) is used for 1 and 2, but a different signing key (F-Droid's) is used for 3. Building a debug APK using method 4 excludes a key entirely. Signing keys help ensure that a user isn't tricked into installing a malicious update to an app. When using method 5, each APK is signed with a different random key supplied by GitHub Actions, so you cannot even update it. You will have to backup and restore the app data each time you wish to use a new APK.
```
Comment (opusforlife2) [inline README.md:100]: "could" -1

Rest of it - +10

File: README.md [hunk 7]
```diff
@@ -97,10 +97,10 @@ You can install NewPipe using one of the following methods:
...
 
 In the meanwhile, if you want to switch sources for some reason (e.g. NewPipe's core functionality breaks and F-Droid doesn't have the latest update yet), we recommend following this procedure:
-1. Back up your data via Settings > Content > Export Database so you keep your history, subscriptions, and playlists
+1. Back up your data via Settings > Content > Export Database, so you keep your history, subscriptions, and playlists
```
Comment (opusforlife2) [inline README.md:103]:
```suggestion
1. Back up your data via Settings > Content > Export Database, so that you keep your history, subscriptions, and playlists
```

**Review by opusforlife2 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (TacoTheDank) [reply README.md]: ^ Agreed

File: README.md (hunk 5 above)
Comment (mhmdanas) [reply README.md:62]: The original sentence is a bit awkward too, I would rephrase it like so:

```suggestion
Also, since both are free and open-source software, neither the app nor the Extractor uses any proprietary libraries or frameworks, such as Google Play Services. This means you can use NewPipe on devices or custom ROMs that do not have Google apps installed.
```

(The second change is unrelated to our discussion)

**Review by mhmdanas — CHANGES_REQUESTED** [review]
I agree with @opusforlife2 on most of his opinions.

File: README.md (hunk 5 above)
Comment (TacoTheDank) [reply README.md:62]: @opusforlife2 According to English grammatical rules, this change would actually be correct.

When referring to both the app and the extractor as "they", it is considered plural. This is because they are being referred to in the collective sense. That is easy.

When using the conjunction "neither/nor," the plurality of the verb in the clause depends on the plurality of at least one of the subjects. For the verb to be plural, at least one of the subjects needs to be plural. As both "app" and "extractor" are singular nouns, the verb would be singular.

Shortened version: Two (or more) singular subjects connected by "neither/nor" take a singular verb. If at least one subject is plural, the verb will be plural. Both "app" and "extractor" are singular, so "uses" would be correct.

Source 1: https://archives.cjr.org/language_corner/either_wins.php
Source 2: https://www.myenglishpages.com/english/grammar-lesson-either-or-neither-nor.php
Source 3: I can thank 8th grade English for making me a grammar buff :)

Comment (TacoTheDank) [reply README.md:62]: Also, I think I prefer using "they" over "both" in this case, as the latter seems a bit awkward.

Better yet, I would suggest using "they are both" or "both of them."

Comment (mhmdanas) [reply README.md:62]: I'm okay with "both of them" too. I'll edit my suggestion.

Comment (opusforlife2) [reply README.md:62]: @TacoTheDank The sources I checked said exactly what you've said, so that part I was fine with. But the sentence as a whole didn't have number agreement, which I didn't find any precedence for.

I'm okay with the edited suggestion.

Comment (mhmdanas) [reply README.md:62]: I don't think the change suggested by the PR is _wrong_ per se, it's just a tad awkward for me (the original sentence is flat-out wrong, as already mentioned).

**Comment by Stypox** [comment]
Closing as there has not been recent activity. Feel free to reopen if you intend to continue working on this PR.
